sys/dev/usb/net/if_axge.c

```diff
@@ -410,6 +412,11 @@ axge_chip_init(struct axge_softc *sc)
 	axge_write_cmd_1(sc, AXGE_ACCESS_MAC, AXGE_CLK_SELECT,
 	    AXGE_CLK_SELECT_ACS | AXGE_CLK_SELECT_BCS);
 	uether_pause(&sc->sc_ue, hz / 10);
+
+	if ((sc->sc_flags & AXGE_FLAG_179A) != 0) {
+		/* Ensure 179A runs in compat 178A/179 fw mode. */
```

Comment:
I think a more detailed comment here would be good. In particular, what do we lose by enabling compat mode?